src/test/java/org/jaudiolibs/jnajack/JackTest.java

```diff
@@ -0,0 +1,81 @@
+package org.jaudiolibs.jnajack;
+
+import com.sun.jna.NativeLong;
+import com.sun.jna.ptr.IntByReference;
+import java.util.EnumSet;
+import org.jaudiolibs.jnajack.lowlevel.JackLibrary;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ *
+ */
+public class JackTest {
+
+    public JackTest() {
+    }
+
+    @BeforeAll
+    public static void setUpClass() {
+    }
+
+    @AfterAll
+    public static void tearDownClass() {
+    }
+
+    @BeforeEach
+    public void setUp() {
+    }
+
+    @AfterEach
+    public void tearDown() {
+    }
+
+    @Test
+    public void testGetTime() throws JackException {
+        JackLibrary lib = mock(JackLibrary.class);
+        long value = 1010101L;
+        when(lib.jack_get_time()).thenReturn(new NativeLong(value));
+        Jack jack = new Jack(lib);
+        assertEquals(value, jack.getTime());
+    }
+
+    @Test
+    public void testOpenClient() throws JackException {
+        JackLibrary lib = mock(JackLibrary.class);
+        int optionsFlag = JackLibrary.JackOptions.JackNoStartServer
+                | JackLibrary.JackOptions.JackUseExactName;
+        int statusFlag = JackLibrary.JackStatus.JackServerStarted
+                | JackLibrary.JackStatus.JackNameNotUnique;
+        String[] inputName = new String[1];
```

Comment:
Thanks. It might be obvious I'm not familiar with Mockito!  For some reason hadn't considered using `verify()` mixed with `when().thenAnswer()`, but this makes things cleaner - have updated.